containers/CSharp/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,74 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/dotnet
+{
+    "name": "C# (.NET)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": { 
+            // Update 'VARIANT' to pick a .NET Core version: 2.1, 3.1, 5.0
+            "VARIANT": "3.1",
+            // Options
+            "NODE_VERSION": "none",
+            "INSTALL_AZURE_CLI": "true"
+        }
+    },
+    "runArgs": [
+        "--network",
+        "host"
+    ],
+
+    "mounts": [
+        // Keep command history 
+        "source=ostf-bashhistory,target=/commandhistory,type=volume",
+        // Use hosts docker socket
+        "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
+    ],
+
+    // Set *default* container specific settings.json values on container create.
+    "settings": {},
+
+    // Add the IDs of extensions you want installed when the container is created.
+    "extensions": [
+        "ms-dotnettools.csharp",
+        "ms-vscode.azure-account",
+        "ms-azuretools.vscode-docker",
+        "ms-vscode.azure-account",
+        "vsciot-vscode.azure-iot-hub",
+        "vsciot-vscode.azure-iot-edge",
+        "vsciot-vscode.azure-iot-toolkit",
+        "vsciot-vscode.azure-iot-tools",
```

Comment:
>iot-tools [](http://example.com/codeflow?start=29&length=9)

this extension already packages iot-hub and iot-edge so we probably don't need both

Reply:
yes - thank you!